sched/semaphore/sem_wait.c

```diff
@@ -71,9 +71,12 @@
 
 int nxsem_wait_slow(FAR sem_t *sem)
 {
-  FAR struct tcb_s *rtcb;
+  FAR struct tcb_s *rtcb = this_task();
```

Comment:
Looks like a redundant code line change

Reply:
nope, this_task is stored here on purpose, it is used in the middle and in the end on nxsem_wait_slow

Reply:
I missed that, thanks